README.md

```diff
@@ -33,6 +33,12 @@ git version 1.7.6
 hub version 2.2.3
 ```
 
+if you want to use the latest unstable version, use:
```

Comment:
1. Please start sentences with a capital letter. That would be "If".
2. A prerelease version is not necessarily "unstable". They pass all CI tests. It's just that their feature set changes often and some flags might be removed or renamed.

     Also, nobody really wants to "use the latest unstable version". If you phrase it like that, that sounds like there is no benefit to using `--devel`, and that there is only a negative consequence: instability. But, the actual benefit is that you gain access to new features quicker. So, let's instead phrase it using a language that reveals a benefit that a user might get from using the `--devel` flag: “If you want to get access to new hub features earlier and help report bugs, you can install the prerelease version”.